docs/guides/rabbitmq/autoscaler/storage/storage-autoscale.md

```diff
@@ -0,0 +1,333 @@
+    Last Transition Time:  2022-01-14T06:14:25Z
+    Message:               Controller has successfully expand the volume of rabbitmq demo/mops-sample-rabbitmq-xojkua
+    Observed Generation:   1
+    Reason:                OpsRequestProcessedSuccessfully
+    Status:                True
+    Type:                  Successful
+  Observed Generation:     3
+  Phase:                   Successful
+Events:
+  Type    Reason      Age    From                        Message
+  ----    ------      ----   ----                        -------
+  Normal  Starting    2m58s  KubeDB Enterprise Operator  Start processing for rabbitmqOpsRequest: demo/mops-sample-rabbitmq-xojkua
+  Normal  Starting    2m58s  KubeDB Enterprise Operator  Pausing rabbitmq databse: demo/sample-rabbitmq
+  Normal  Successful  2m58s  KubeDB Enterprise Operator  Successfully paused rabbitmq database: demo/sample-rabbitmq for rabbitmqOpsRequest: mops-sample-rabbitmq-xojkua
+  Normal  Successful  103s   KubeDB Enterprise Operator  Volume Expansion performed successfully in rabbitmq pod for rabbitmqOpsRequest: demo/mops-sample-rabbitmq-xojkua
+  Normal  Starting    103s   KubeDB Enterprise Operator  Updating rabbitmq storage
+  Normal  Successful  103s   KubeDB Enterprise Operator  Successfully Updated rabbitmq storage
+  Normal  Starting    103s   KubeDB Enterprise Operator  Resuming rabbitmq database: demo/sample-rabbitmq
+  Normal  Successful  103s   KubeDB Enterprise Operator  Successfully resumed rabbitmq database: demo/sample-rabbitmq
+  Normal  Successful  103s   KubeDB Enterprise Operator  Controller has Successfully expand the volume of rabbitmq: demo/sample-rabbitmq
+```
+
+Now, we are going to verify from the `Petset`, and the `Persistent Volume` whether the volume of the replicaset database has expanded to meet the desired state, Let's check,
+
+```bash
+$ kubectl get sts -n demo sample-rabbitmq -o json | jq '.spec.volumeClaimTemplates[].spec.resources.requests.storage'
+"1594884096"
+$ kubectl get pv -n demo
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                        STORAGECLASS          REASON   AGE
+pvc-43266d76-f280-4cca-bd78-d13660a84db9   2Gi        RWO            Delete           Bound    demo/data-sample-rabbitmq-2   topolvm-provisioner            23m
+pvc-4a509b05-774b-42d9-b36d-599c9056af37   2Gi        RWO            Delete           Bound    demo/data-sample-rabbitmq-0   topolvm-provisioner            24m
+pvc-c27eee12-cd86-4410-b39e-b1dd735fc14d   2Gi        RWO            Delete           Bound    demo/data-sample-rabbitmq-1   topolvm-provisioner            23m
+```
+
+The above output verifies that we have successfully autoscaled the volume of the rabbitmq replicaset database.
+
+## Cleaning Up
+
+To clean up the Kubernetes resources created by this tutorial, run:
+
+```bash
+kubectl delete rabbitmq -n demo sample-rabbitmq
+kubectl delete rabbitmqautoscaler -n demo md-as-st
```

Comment:
need to fix this type of naming problem `md-as-st`. No autoscaler created with this name